src/ProductBundle/Resources/skeleton/product/Provider/EntityProductProvider.php

```diff
@@ -26,6 +27,11 @@ class {{ product }}ProductProvider extends BaseProductProvider
 {
     public function getBaseControllerName()
     {
-        return 'SonataProductBundle:{{ product }}';
+        return {{ product }}Controller::class;
+    }
+
+    public function getTemplatesPath()
```

Comment:
You will have to add the return type hint here and below
```suggestion
    public function getTemplatesPath(): string
```